packages/vue-cli-plugin-typescript/generator/template/src/router/index.ts

```diff
@@ -25,11 +25,16 @@ const routes: Array<RouteConfig> = [
       if (!isEnvValid()) {
         next('/error');
       } else {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const res = await fetch(process.env.VUE_APP_TOKEN_ENDPOINT!);
```

Comment:
putting an eslint-disable is a last resort solution always.
We should comply with the rule imposed on us to the best of our ability.

See my comment on envUtils for guidance ;)